helpers/Upload.js

```diff
@@ -5,7 +5,7 @@ const upload = async (posts) => {
     await Fetch(`${process.env.SCRAPER_ADMIN_URL}/upload`, {
       headers: {
         'Content-Type': 'application/json',
-        'scraper-base-url': process.env.SCRAPER_BASE_URL,
+        'scraper-base-url': `${process.env.SCRAPER_BASE_URL}`,
```

Comment:
no need to use template string in this case. replace with this `'scraper-base-url': process.env.SCRAPER_BASE_URL,`